SU-SWS/stanford_bean_types
# 7.x 2.x dev soe 3382 hide file insert

# NOT READY FOR REVIEW

# Summary
_See ticket:_ https://stanfordits.atlassian.net/browse/SOE-3382

# Needed By (Date)
- Sprint end
_For version release: https://stanfordits.atlassian.net/projects/SOE/versions_

# Criticality
- Fixes broken stuff


# Steps to Test
- Checkout this branch: `git checkout`
- May need to revert feature
- Navigate to __Path__
- Verify you see __Text__

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)

## Related PRs
https://stanfordits.atlassian.net/browse/SOE-3382
## More Information

## Folks to notify
`@minorwm `
`@annawatt `
`@kerri-augenstein `
`@boznik `
`@cjwest`

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
